lib/src/components/EditorFieldTemplate/index.tsx

```diff
@@ -34,18 +32,13 @@ const messages = defineMessages({
 export default function EditorFieldTemplate({ id, schema, children }: FieldTemplateProps) {
   const [isEditing, setIsEditing] = useState(false);
   const verna = useVerna();
-  const DropZone = verna.DropZone;
   const path = id.split(RJSF_ID_SEPARATOR);
   const isRoot = id === 'root';
   const isSection = path.length === 2 && !verna.selector;
   const ownProperties = Object.keys(schema.properties || {}).length > 0;
   const canAddField = schema.type !== 'object';
   const canAddSection = isSection && !isRoot && !verna.selector;
-
-  function addItem(widgetProps?: ShowCaseWidgetProps) {
-    const newKey = uuidv4();
-    addWidget(newKey, id, verna, widgetProps);
-  }
```

Comment:
Hmmm... this is the only accessible way we had to add an input. Not sure it is a good idea to remove it at this point nope ?

Reply:
Well it is if we don't use the drag n drop, but the feature itself isn't finished well, it create a default widget of type "textWidget", not sure if it is really relevant to have it